src/translator_testing_model/schema/translator_testing_model.yaml

```diff
@@ -0,0 +1,479 @@
+---
+id: https://w3id.org/TranslatorSRI/test-case-schema
+name: Translator-Testing-Model
+title: Translator Testing Data Model
+description: |-
+  Data model to formalize the structure of test assets, cases, suites and related metadata
+  applied to run the diverse polymorphic testing objectives for the Biomedical Data Translator system.
+license: MIT
+see_also:
+  - https://TranslatorSRI.github.io/test-case-schema
+
+prefixes:
+  ttm: https://w3id.org/TranslatorSRI/Translator-Testing-Model/
+  linkml: https://w3id.org/linkml/
+  biolink: https://w3id.org/biolink/
+  schema: http://schema.org/
+  example: https://example.org/
+
+default_prefix: ttm
+default_range: string
+
+imports:
+  - linkml:types
+
+classes:
+
+  TestEntity:
+    description: >-
+      Parent identification class of all major classes within the data model for Translator testing.
+    abstract: true
+    tree_root: true
+    slots:
+      - id
+      - name
+      - description
+
+  TestMetadata:
+    description: >-
+      Represents metadata related to (external SME, SMURF, Translator feedback, 
+      large scale batch, etc.) provenance of test assets, cases and/or suites.
+    is_a: TestEntity
+    slots:
+      - test_source
+      - test_reference
+      - test_objective
+
+  TestAsset:
+    description: >-
+      Represents a Test Asset, which is a test case agnostic
+      semantic specification of a Translator test target.
+    is_a: TestEntity
+    slots:
+      - input_id
+      - input_name
+      - output_id
+      - output_name
+      - expected_output
+      - test_issue
+      - semantic_severity
+      - in_v1
+      - well_known
+    slot_usage:
+      id:
+        aliases: ["AssetIdentifier"]
+        range: uriorcurie
+
+  QueryAnswerPair:
+    description: >-
+      Represents a QueryAnswerPair specification of a Test Asset
+#    aliases: [""]
+    is_a: TestAsset
+  
+  TestEdgeData:
+    description: >-
+      Represents a single Biolink Model compliant instance of an edge that can be used for testing.
+    is_a: TestAsset
+
+  TestCase:
+    description: >-
+      Represents a static specification of a Test Case
+    is_a: TestEntity
```

Comment:
Can a TestCase be provided via an `input_id` to a TestAssiet

Reply:
same question for TestSuite - maybe we need a more granular range on the `inputs` slot here via slot_usage?  And/or some understanding of what classes can be represented by uriorcuries in the `input_id` slot?

Reply:
This feels like it violates the model hierarchical relationship between a **TestAsset** and a **TestCase** (typically created from a **TestAsset**). That said, as the need arises, we could specify a mechanism for reverse engineering ("extracting") a previously undocumented **TestAsset** from a given **TestCase**. I suppose if this derivation is useful to record, we could add another slot to the model. I wouldn't, however, call it an input per say. Inputs are much more granular in nature.

That said, I'll take another look at this in a few minutes.

I trust that my comments above have answered most of your questions about the PR.